apps/src/constants.js

```diff
@@ -44,6 +44,7 @@ exports.TestResults = {
 
   // start using negative values, since we consider >= 20 to be "solved"
   EMPTY_FUNCTION_NAME: -1,       // We have a variable or function with the name ""
+  NESTED_FOR_SAME_VARIABLE: -2,
```

Comment:
From my perspective, the sooner we separate them, the better. As long as the historical overlap is sufficiently documented, separating them now will yield better day in the future.